starlite/asgi.py

```diff
@@ -20,6 +29,18 @@
     from starlite.types import LifeCycleHandler
 
 
+# Sentinel object to represent a path param in the route map
+class _PathParam:
```

Comment:
some points here-

1. no need for the leading underscore IMO
2. I would rename it to PathParamPlaceholder or something such like
3. comment should be a docstring
4. whats the point of having a repr method?

You can simply use the class, no need to use an instance for it. See usage for example of `EmptyType` and `Empty` from `starlite.types`.

Reply:
Using an instances allows the repr to show something other than `<class 'starlite.asgi.PathParamPlaceholder'>` when being inspected in a debugger, in this case a * which is what the original implementation used to denote a param placeholder. But just referencing the class directly works just as well.